validator/main.py

```diff
@@ -99,27 +99,28 @@ def get_llm_response(self, prompt: str) -> str:
         # 3. Return the response
         return response
 
-    def validate(self, value: Any, metadata: Dict) -> ValidationResult:
+    def validate(self, value: str, metadata: Dict) -> ValidationResult:
         """
         Validates is based on the relevance of the reference text to the original question.
 
         Args:
-            value (Any): The value to validate. It must contain 'original_prompt' and 'reference_text' keys.
-            metadata (Dict): The metadata for the validation. This is not used in the current implementation.
+            value (str): The value to validate.
+            metadata (Dict): The metadata for the validation. It must contain the key 'original_prompt', 
+            with the original question that the reference text is being compared to.
 
         Returns:
             ValidationResult: The result of the validation. It can be a PassResult if the reference 
                               text is relevant to the original question, or a FailResult otherwise.
         """
         # 1. Get the question and arg from the value
-        original_prompt = value.get("original_prompt")
+        original_prompt = metadata.get("original_prompt")
         if original_prompt is None:
             raise RuntimeError(
                 "original_prompt missing from value. "
                 "Please provide the original prompt."
             )
 
-        reference = value.get("reference_text")
+        reference = value
         if reference is None:
             raise RuntimeError(
                 "'reference_text' missing from value. "
```

Comment:
should these be RuntimeErrors or UserFacingExceptions? cc @CalebCourier

Reply:
Just looked at where we differentiate and, whether intentional or not, it doesn't matter anymore because we treat them all the same now: https://github.com/guardrails-ai/guardrails/blob/main/guardrails/run/runner.py#L235-L242

That said the error message for this line should be updated to reflect the new source of `reference`.

Reply:
In other validators we appear to be using `ValueError`, but I think `RuntimeError` is actually more appropriate here since we're only performing a type check (not `None`) and not asserting anything on the actual value.